README.md

```diff
@@ -2,6 +2,53 @@
 
 A NestJS module for managing environment variables easily and securely.
 
+<hr/>
+
+## What Difference From [official library](https://docs.nestjs.com/techniques/configuration)
```

Comment:
It seems like it doesn't resolved well 🤔
Currently, this section is under `## Features`.

Reply:
oops!. sorry, could you please check at it again?